<h2>Description</h2>
<p>This hub hosts assemblies of several <em>Saccharomyces cerevisiae</em> strains belonging to the <strong>Peterhof genetic collection</strong>
 and SNV tracks relative to the sacCer3 genome assembly.</p>

<p> Data for the following genomes are available at this page:

<h3> Saccharomyces cerevisiae assembly (sacCer3/) </h3>
<p>These tracks contain information about single nucleotide variations in several  <em>Saccharomyces cerevisiae</em> strains of the Peterhof genetic collection as well as some other strains related to this project.
For each strain, there are two separate tracks, one for substituions (SNV) and the other for short indels. For core strains of the collection, there are also tracks showing copy number variation.</p>
<p> SNV data are available for the following strains: 15V-P4, 25-25-2V-P3982, 1B-D1606, 74-D694, 6P-33G-D373, A1-K5-35B-D924, nm126-A1-K5-35B-D924, YPH499, and D273-10B.  </p>

<a name="15V"> </a>
<h3> 15V-P4 (15V_contigs/ and 15V_scaffolds/) </h3>

<p> This part of the hub shows assembly of the 15V-P4 strain, one of the progenitor strains for the whole collection. </p>

<a name="25b"> </a>
<h3> 25-25-2V-P3982 (25b_contigs/ and 25b_scaffolds/) </h3>

<p> This part shows assembly of the 25-25-2V-P3982 strain, a Peterhof strain with a long history of laboratory breeding. </p>

<a name="1B"> </a>
<h3> 1B-D1606 (1B_contigs/ and 1B_scaffolds) </h3>

<p> This part of the hub features assembly of 1B-D1606, a hybrid strain descending to both Peterhof and S288C-related collections of yeast strains. </p>

<a name="74"> </a>
<h3> 74-D694 (74_contigs/ and 74_scaffolds/) </h3>

<p> This part shows assembly of 74-D694, another hybrid strain, which has been widely used to study yeast prions. </p>

<a name="test"> </a>
<h3> <i> S. cerevisiae </i>  15V-P4 (test) (compare_annotations/) (ex-15V_contigs/) </h3>

<p> This part of the hub collects (and allows one to compare) sets of annotations derived with different algorithms (listed as track names) as of May 2015. 
It is based on a slightly different assembly than the "final" 15V-P4 assembly, so contig names may be different.
This work was done as a project in the Bioinformatics Institute <a href="http://bioinformaticsinstitute.ru/"> http://bioinformaticsinstitute.ru/</a>. 
For more information you might want to look <a href="http://bioinformaticsinstitute.ru/node/982"> here [Russian]</a>. 
</p>
<p> This part mostly serves educational purposes and is unlikely to be updated. </p>

<h2>Methods</h2>

<p> Genomes were assembled <i> de novo </i> with SPAdes v3.1 or v3.6 (applies to the 74-D694 genome only) and scaffolded with <a href="https://github.com/gtamazian/chromosomer"> chromosomer</a>.
Comprehensive sets of genome annotations were created with Maker2. </p>

<p>For SNV analysis, short reads were aligned with bowtie v2.1.0; SNV calling was performed with samtools v1.0 mpileup. SNVs with quality less than 30 or DP 
less than 3, as well as heterozygous indels,  were filtered out. The resulting files were annotated with SnpEff v4.1. </p>

<h2> Usage notes </h2>

<p> Contig naming conventions: NODE_int_int (number and ID). Scaffolds: I to XVI. </p>


<h2> References </h2>
<p> Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods; 2012 Apr;9(4):357. Available from: <a href=http://dx.doi.org/10.1038/nmeth.1923> http://dx.doi.org/10.1038/nmeth.1923 </a>
</p>
<p> Cingolani P, Platts A, Wang LL, Coon M, Nguyen T, Wang L, et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. Fly (Austin); 2012;6(2):80-92. Available from: <a href=https://www.landesbioscience.com/journals/fly/article/19695/> https://www.landesbioscience.com/journals/fly/article/19695/ </a> 
</p>
<p> Li H. A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics [Internet]. 2011 Nov 1 [cited 2015 Mar 6];27(21):2987-93. Available from: <a href=http://bioinformatics.oxfordjournals.org/content/27/21/2987.long> http://bioinformatics.oxfordjournals.org/content/27/21/2987.long </a> </p>
<p> Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, et al. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol; 2012; 19(5):455-77. Available from: <a href=http://online.liebertpub.com/doi/abs/10.1089/cmb.2012.0021> http://online.liebertpub.com/doi/abs/10.1089/cmb.2012.0021 </a> </p>
<p> Holt C, Yandell M. MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects. BMC Bioinformatics [Internet]. 2011 Jan;12(1):491. Available from: <a href=http://www.biomedcentral.com/1471-2105/12/491> http://www.biomedcentral.com/1471-2105/12/491 </a> </p>



<h2> Contacts </h2>
If you have questions please contact Polina Drozdova at drozdovapb@gmail.com or drozdovapb@bio.isu.ru

