<h2>Description</h2>
<p>This hub hosts assemblies of several <em>Saccharomyces cerevisiae</em> strains belonging to the <strong>Peterhof genetic collection</strong>
 and SNV tracks relative to the sacCer3 genome assembly.</p>

<p> Data for the following genomes are available at this page:

<h3> Saccharomyces cerevisiae assembly (sacCer3/) </h3>
<p>These tracks contain information about single nucleotide variations in several  <em>Saccharomyces cerevisiae</em> strains of the Peterhof genetic collection.
For each strain, there are two separate tracks, one for substituions (SNV) and the other for short indels. </p>

<h3> 15V-P4 (15V_contigs/ and 15V_scaffolds/) </h3>

<p> This part of the hub shows assembly of the 15V-P4 strain, one of the progenitor strains for the whole collection. </p>

<h3> 25-25-2V-P3982 (25b_contigs/ and 25b_scaffolds/) </h3>

<p> This part shows assembly of the 25-25-2V-P3982 strain, a Peterhof strain with a long history of laboratory breeding. </p>

<h3> 1B-D1606 (1B_contigs/ and 1B_scaffolds) </h3>

<p> This part of the hub features assembly of 1B-D1606, a hybrid strain descending to both Peterhof and S288C-related collections of yeast strains. </p>

<h3> 74-D694 (74_contigs/ and 74_scaffolds/) </h3>

<p> This part shows assembly of 74-D694, one more hybrid strains, which has been widely used to study yeast prions. </p>


<h3> <i> S. cerevisiae </i>  15V-P4 (test) (compare_annotations/) (ex-15V_contigs/) </h3>
<p> This part of the hub collects (and allows one to compare) sets of annotations derived with different algorithms (listed as track names) as of May 2015. 
It is based on a slightly different assembly than the "final" 15V-P4 assembly, so contig names may be different.
This work was done as a project in the Bioinformatics Institute <a href="http://bioinformaticsinstitute.ru/"> http://bioinformaticsinstitute.ru/</a>. 
For more information you might want to look <a href="http://bioinformaticsinstitute.ru/node/982"> here [Russian]</a>. 
</p>
<p> This part mostly serves educational purposes and is unlikely to be updated. </p>

<h2>Methods</h2>

<p> Genomes were assembled <i> de novo </i> with SPAdes v3.1 or v3.6 (applies to the 74-D694 genome only) and scaffolded with <a href="https://github.com/gtamazian/chromosomer"> chromosomer </a>.
Comprehensive sets of genome annotations were created with Maker2. </p>

<p>For SNV analysis, short reads were aligned with bowtie v2.1.0; SNV calling was performed with samtools v1.0 mpileup. SNVs with quality less than 30 or DP 
less than 3, as well as heterozygous indels,  were filtered out. The resulting files were annotated with SnpEff v4.1. </p>

<h2> Usage notes </h2>

<p> Contig naming conventions: NODE_int_int (number and ID). Scaffolds: I to XVI. </p>


<h2> Contacts </h2>
If you have questions please contact Polina Drozdova at <a href="mailto:p.drozdova@bio.spbu.ru"> p.drozdova<at>bio.spbu.ru </a>.
Please note that the data can be accessed at <a href="https://github.com/drozdovapb/PeterhofYeastHub"> on github </a>.

