<h2>Description</h2>
<p>These tracks contain information about single nucleotide variations in several  <em>Saccharomyces cerevisiae</em> strains of the Peterhof genetic collection. 
For more information on strains please see <a href=http://public.dobzhanskycenter.ru/PeterhofYeastHub/hub_contents.html> the description page</a>.


<h2>Methods</h2>
Short reads were aligned with bowtie v2.1.0; SNV calling was performed with samtools v1.0 mpileup. SNVs with quality less than 30 or DP less than 3, 
as well as indels, were filtered out. The resulting files were annotated with SnpEff v4.1.

<h2>References</h2>

<p> Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods; 2012 Apr;9(4):357-9. Available from: <a href=http://dx.doi.org/10.1038/nmeth.1923> http://dx.doi.org/10.1038/nmeth.1923 </a>
</p>
<p> Cingolani P, Platts A, Wang LL, Coon M, Nguyen T, Wang L, et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. Fly (Austin); 2012 Jan 1;6(2):80-92. Available from: <a href=https://www.landesbioscience.com/journals/fly/article/19695/> https://www.landesbioscience.com/journals/fly/article/19695/ </a> 
</p>
<p> Li H. A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics; 2011 Nov 1;27(21):2987-93. Available from: <a href=http://bioinformatics.oxfordjournals.org/content/27/21/2987.long> http://bioinformatics.oxfordjournals.org/content/27/21/2987.long </a> </p>

<h2> Contacts </h2>
If you have questions please contact Polina Drozdova at <a href="mailto:p.drozdova@spbu.ru"> p.drozdova@spbu.ru</a>.




